README.md

```diff
@@ -30,3 +30,25 @@ Additionally, this allows you to store the title semantically, in the document i
   gems:
     - jekyll-titles-from-headings
   ```
+
+## Configuration
+If your theme renders titles based on `page.title`, you can remove the title from the content by by setting
+
+```yml
+strip_title: true
```

Comment:
What do you think about namespacing the setting in case (A) we want to add another setting and (B) to avoid namespace conflicts?